### DISCUSSION TITLE
Can we get a new minor release?

### DISCUSSION THREAD

**Discussion by agadzik** [jianyuan/terraform-provider-sentry#448] [General]
Hello!

We're eagerly awaiting a new release that includes https://github.com/jianyuan/terraform-provider-sentry/pull/441.  Is there any chance we can get a new release asap?

Thanks for your time and help, it's much appreciated!

cc @jianyuan

**Comment by jianyuan** [comment]
@agadzik I plan to ship a new release within a day or two.

**Comment by jianyuan** [comment]
[v0.13.0](https://github.com/jianyuan/terraform-provider-sentry/releases/tag/v0.13.0) has been released!
